src/tslint.ts

```diff
@@ -86,6 +86,21 @@ class Linter {
     private containsRule(rules: Lint.RuleFailure[], rule: Lint.RuleFailure) {
         return rules.some((r) => r.equals(rule));
     }
+
+    private getRulesDirectories(): Array<string> {
```

Comment:
let's make this a `public static` method that takes `rulesDirectory: string | string[]` as an argument. then it can be reused in tslint-cli.ts.